queries.sql

```diff
@@ -1,3 +1,41 @@
--- /*Queries that provide answers to the questions from all projects.*/
+-- Find all animals whose name ends in "mon".
+SELECT * 
+FROM animals
+WHERE name LIKE '%mon'
 
--- SELECT * from animals WHERE name =   'Luna';
+-- List the name of all animals born between 2016 and 2019.
+SELECT name
+FROM animals
+WHERE (date_of_birth) between '2016-01-01' and '2019-12-31'
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+
+SELECT name
+FROM animals
+WHERE neutered=true and escape_attempts <3
```

Comment:
- [ OPTIONAL ] I think it would be great if you add quotes to the value since each value surronded with quotes so kindly add quotes suuround true value and number 3 value to be somthing like this 🤩✨🔥💯

> SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < '3';